### PR TITLE
Remove empty paragraph in Contact Information pattern

### PR DESCRIPTION
**Pull request by melchoyce**
## Summary
There is an empty paragraph block in the Contact Information pattern. This PR removes it.

## Screenshots

Before:

image

After:

image

### CODE REVIEW

File: inc/block-patterns.php [hunk 1]
```diff
@@ -114,7 +114,7 @@
...
...v class="wp-block-column"><!-- wp:paragraph {"align":"center"} --><p class="has-text-align-center">' . esc_html_x( '123 Main Street', 'Block pattern sample content', 'twentytwentyone' ) . '<br>' . esc_html_x( 'Cambridge, MA, 02139', 'Block pattern sample content', 'twentytwentyone' ) . '</p><!-- /wp:paragraph --></div><!-- /wp:column --><!-- wp:column {"verticalAlignment":"center"} --><div class="wp-block-column is-vertically-aligned-center"><!-- wp:social-links {"align":"right","className":"is-style-twentytwentyone-social-icons-color"} --><ul class="wp-block-social-links alignright is-style-twentytwentyone-social-icons-color"><!-- wp:social-link {"url":"https://wordpress.org","service":"wordpress"} /--><!-- wp:social-link {"url":"https://www.facebook.com/WordPress/","service":"facebook"} /--><!-- wp:social-link {"url":"https://twitter.com/WordPress","service":"twitter"} /--><!-- wp:social-link {"service":"instagram"} /--><!-- wp:social-link {"service":"linkedin"} /--><!-- wp:social-link {"service":"youtube"} /--><!-- wp:social-link {"url":"https://www.youtube.com/wordpress","service":"youtube"} /--></ul><!-- /wp:social-links --></div><!-- /wp:column --></div><!-- /wp:columns -->',
```
Comment (ryelle) [inline inc/block-patterns.php]:
```suggestion
			'content'     => '<!-- wp:columns {"align":"wide"} --><div class="wp-block-columns alignwide"><!-- wp:column --><div class="wp-block-column"><!-- wp:paragraph --><p><a href="mailto:#">' . esc_html_x( 'example@example.com', 'Block pattern sample content', 'twentytwentyone' ) . '<br></a>' . esc_html_x( '123-456-7890', 'Block pattern sample content', 'twentytwentyone' ) . '</p><!-- /wp:paragraph --></div><!-- /wp:column --><!-- wp:column --><div class="wp-block-column"><!-- wp:paragraph {"align":"center"} --><p class="has-text-align-center">' . esc_html_x( '123 Main Street', 'Block pattern sample content', 'twentytwentyone' ) . '<br>' . esc_html_x( 'Cambridge, MA, 02139', 'Block pattern sample content', 'twentytwentyone' ) . '</p><!-- /wp:paragraph --></div><!-- /wp:column --><!-- wp:column {"verticalAlignment":"center"} --><div class="wp-block-column is-vertically-aligned-center"><!-- wp:social-links {"align":"right","className":"is-style-twentytwentyone-social-icons-color"} --><ul class="wp-block-social-links alignright is-style-twentytwentyone-social-icons-color"><!-- wp:social-link {"url":"https://wordpress.org","service":"wordpress"} /--><!-- wp:social-link {"url":"https://www.facebook.com/WordPress/","service":"facebook"} /--><!-- wp:social-link {"url":"https://twitter.com/WordPress","service":"twitter"} /--><!-- wp:social-link {"service":"instagram"} /--><!-- wp:social-link {"service":"linkedin"} /--><!-- wp:social-link {"url":"https://www.youtube.com/wordpress","service":"youtube"} /--></ul><!-- /wp:social-links --></div><!-- /wp:column --></div><!-- /wp:columns -->',
```

I noticed two youtube icons, and only one of them has a URL, this code removes the empty one. It looks like linkedin and instagram are also empty, so they don't show up on the frontend. Are they meant to be placeholders?

**Review by ryelle — COMMENTED** [review]
👍  Removes the paragraph, but I noticed the icons are a little odd (see suggestion)

File: inc/block-patterns.php (hunk 1 above)
Comment (melchoyce) [reply inc/block-patterns.php]: If the social block was updated in core, would that impact the pattern? Wondering if it came from a Gutenberg update.

Comment (ryelle) [reply inc/block-patterns.php]: I don't _think so,_ it looks like it probably never had a URL. You can see some of them have `"url":"https://www.youtube.com/wordpress"` in the `{}`, but the instagram one only has `<!-- wp:social-link {"service":"instagram"} /-->`.

Comment (melchoyce) [reply inc/block-patterns.php]: Okay — we should remove those too 👍

Comment (ryelle) [reply inc/block-patterns.php]: done! Removed the 3 empty link blocks: instagram, linkedin, and the extra youtube.

**Pull request merged**